arroyo/backends/kafka/consumer.py

```diff
@@ -566,16 +575,22 @@ def paused(self) -> Sequence[Partition]:
 
         return [*self.__paused]
 
-    def stage_positions(self, positions: Mapping[Partition, Position]) -> None:
+    def stage_positions(
+        self, positions: Mapping[Partition, Position], force: bool = False
```

Comment:
The force flag here is only used for the tests to commit a nonsense offset.

Reply:
Could you please explain this in the docstring ?